scrapy_poet/middleware.py

```diff
@@ -39,20 +43,33 @@ def process_request(self, request: Request, spider: Spider):
     @inlineCallbacks
     def process_response(self, request: Request, response: Response,
                          spider: Spider):
-        """This method instantiates all ``Injectable`` sub-classes declared as
+        """This method instantiates all ``Injectable`` subclasses declared as
```

Comment:
It might be also interesting to mention what happens when there is a collision between an already set `cb_kwargs` and an injectable attribute: User-defined cb_kwargs takes precedence. 

See https://github.com/scrapinghub/scrapy-poet/pull/30/files#diff-49772e5c5730b71f0e27fcc38b4f9614R80

Reply:
Done.